### PR TITLE
Add a new docker-compose configuration with the wallet-proxy and the identity provider service.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Make it possible for partners to test in an isolated chain/environment.

## Changes

Change the broken docker-compose configuration with the wallet proxy into a working configuration that also exposes the identity provider service.

This is not the finished product, we would want to add more documentation and also the middleware, the node and network dashboards, but that is more work since useful images are not readily available, and I do not have the time for it now.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [ ] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: docker-compose/baker+wallet-proxy+identity-provider.yaml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      - db-volume:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: concordium
+      POSTGRES_USER: concordium
+      POSTGRES_DB: concordium
+    ports:
+    - 5432:5432
+
+  wallet_proxy:
+    image: abizjak/wallet-proxy:latest
```
Comment (abizjak) [inline docker-compose/baker+wallet-proxy+identity-provider.yaml:83]: This is currently using my own space until images are posted publicly under concordium.

File: docker-compose/baker+wallet-proxy+identity-provider.yaml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    volumes:
+      # path to genesis data, relative to this file, or absolute path
+      - $HOST_ROOT/configuration/wallet-proxy:/data
+    entrypoint:
+    - /start.sh
+    ports:
+    - 14000:3000
+
+  identity_verifier:
+    image: abizjak/test-identity-provider:latest
```
Comment (abizjak) [inline docker-compose/baker+wallet-proxy+identity-provider.yaml:112]: This is currently using my own space until images are posted publicly under concordium.

**Comment by bisgardo** [comment]
Sorry I completely missed this PR. I'll be happy to look into continuing this work once I'm done with my current fixes to the setup.

**Comment by abizjak** [comment]
I did this in a rush before mainnet launch since some partners needed it. I'm not sure how much of it can or should be reused. It definitely needs more polish.

**Pull request closed without merging**